app/models/page_sequence.rb

```diff
@@ -27,7 +27,7 @@ def next_slug
 
   def previous_slug
     slug_index = current_slug_index
-    dead_end_slugs = %w[complete existing-session eligible-now eligibility-confirmed eligible-later ineligible]
+    dead_end_slugs = %w[complete existing-session eligible-now eligible-later ineligible]
```

Comment:
Currently these slugs are dead-ends in that you shouldn't see a back button. If we're going to allow people to change their answers, this might not be the right thing to do any longer.